CheatSheets/chrome-devtools-cheatsheet.md

```diff
@@ -17,88 +17,131 @@
 
 ## Keyboard shortcuts for opening DevTools
 
-| Command | Description |
-| ------- | ----------- |
-| `Ctrl + Shift + J` | Open the ___Console___ panel |
+| Action | Mac  | Windows/Linux  | 
+| :----: | :--: | :------------: |
+| Open the ___Console___ panel | `cmd(⌘)+option(⌥)+j` | `ctrl+shift+j` | 
```

Comment:
```suggestion
| Windows / Linux | Mac | Description |
| ------- | ----------- | ---------- |
| `Ctrl + Shift + J` | `cmd(⌘) + option(⌥) + J` | Open the ___Console___ panel |
```
## Keyboard shortcuts for opening DevTools

| Windows / Linux | Mac | Description |
| ------- | ----------- | ---------- |
| `Ctrl + Shift + J` | `cmd(⌘) + option(⌥) + J` | Open the ___Console___ panel |

___Modification Requirements:___
- Use `Ctrl + Shift + J` instead of `ctrl+shift+j` based on keyboard (___Indentation & Capitalization___)
- Follow the above table format [C1, C2, C3 > Windows / Linux, Mac, Description / Action]

I appreciate your work! Looks to me good!

Reply:
Will modify the code. Apologies for the delay. 

For the screenshot I used the Chrome Extension GoFullPage 
(https://chrome.google.com/webstore/detail/gofullpage-full-page-scre/fdpohaocaechififmbbbbbknoalclacl)[https://chrome.google.com/webstore/detail/gofullpage-full-page-scre/fdpohaocaechififmbbbbbknoalclacl]

Reply:
@crescentpartha I have updated the format for this page. Would you like me to revert this PR so the changes take effect? Or should I do another PR? Just want to make sure I update the page as expected.

Reply:
Create another PR, If possible.

Reply:
Cool. Submitted #213

Reply:
> Will modify the code. Apologies for the delay.
> 
> For the screenshot I used the Chrome Extension GoFullPage https://chrome.google.com/webstore/detail/gofullpage-full-page-scre/fdpohaocaechififmbbbbbknoalclacl
> 
> It was recommended to me from another open-source project I worked on. It has been a very useful extension.

Thank you so much (@abuna1985) for sharing this useful chrome extension. After that, I created an `extension-cheatsheet.md`😊. I think It might be useful for developers.